src/realm/object-store/class.hpp

```diff
@@ -32,20 +32,50 @@ class Query;
 
 class Class {
 public:
-    Class(const std::shared_ptr<Realm>& r, StringData object_type);
+    Class(std::shared_ptr<Realm> r, StringData object_type);
+    Class(std::shared_ptr<Realm> r, const ObjectSchema* object_schema);
+
+    size_t nb_objects() const
```

Comment:
Why `nb_objects()` rather than `size`? I don't think I've seen `nb` used like this before, usually `n_foos` or `num_foos` prefixes for things that return the number of foos.

Reply:
I just felt that a class could not itself have a size, but rather a number of objects could be of that class. I can change to num_objects.